rules/windows/image_load/image_load_susp_vss_dll_load.yml

```
@@ -0,0 +1,26 @@
+title: Image Load of VSS Dll by Uncommon Executable
+id: 37774c23-25a1-4adb-bb6d-8bb9fd59c0f8
+status: experimental
+description: Detects the image load of vss dll by uncommon executables
+references:
+    - https://github.com/ORCx41/DeleteShadowCopies
+author: frack113
+date: 2022/10/31
+tags:
+    - attack.defense_evasion
+    - attack.impact
+    - attack.t1490
+logsource:
+    category: image_load
+    product: windows
+detection:
+    selection:
+        ImageLoaded|endswith: 
+            - '\vsstrace.dll'
+            - '\vssapi.dll'
+    filter_legit:
+        Image|contains: 'c:\windows\'
```

I think it's better to filter out legit software in Program Files too
```suggestion
    filter_legit:
        Image|startswith:
            - 'C:\Windows\'
            - 'C:\Program Files\'
            - 'C:\Program Files (x86)\
```